rfc/0004-at_plus.md

```diff
@@ -195,7 +195,14 @@ The following user errors need to be covered by the SDKs:
     ms.add_results(*results)  # A `Result` is the Python equivalent of a Document
     q.consistent_with(ms)
     ```
- - **PHP:** *not included*
+ - **PHP:**
+
+    ```php
+    $ms = new MutationState()
```

Comment:
that still doesn't align with the methods specified above, right? No static from methods and "addResults" instead of add"

Not sure why python diverges here, but lets try to be consistent if we can :)